manifest_staging/charts/eraser/values.yaml

```diff
@@ -43,3 +43,4 @@ scanner:
     args: []
 
 nameOverride: ""
+eraserPullSecretName: ""
```

Comment:
should we do this per image instead of global so it'll be more future proof

Reply:
do we want it a list similar to https://github.com/Azure/azure-workload-identity/blob/main/charts/workload-identity-webhook/values.yaml#L13

Reply:
It's a pod-level setting (can't be done per image if the containers share a pod), so collector/scanner/eraser will always have the same set of pullSecrets. If necessary we could let the user set a different secret for the manager.

Reply:
We can do a list instead, yes. It makes the logic a little more complicated because those values have to make their way to the code that generates the collector, etc pods.

Reply:
Just pushed changes which make this a list of objects `{ "name": "mysecret" }`